setup.py

```diff
@@ -20,14 +20,14 @@
 ]
 
 setup(
-    name='pyjose',
+    name='jose',
```

Comment:
I don't like to compete with two other projects over the namespace. Let's find another name.